docs/manuals/source/Appendix/Troubleshooting.rst

```diff
@@ -252,7 +218,7 @@ Do not proceed to the next item until you have succeeded with the previous one.
 #. Make sure you have a valid and correct Device resource corresponding to your drive. For Linux users, generally, the default one works. For FreeBSD users, there are two possible Device configurations (see below). For other drives and/or OSes, you will need to first ensure that your system tape modes are properly setup (see below), then possibly modify you Device resource depending on the output from the btape program (next item). When doing this, you should consult the
    :ref:`Storage Daemon Configuration <StoredConfChapter>` of this manual.
 
-#. If you are using a Fibre Channel to connect your tape drive to Bareos, please be sure to disable any caching in the NSR (network storage router, which is a Fibre Channel to SCSI converter).
+#. If you are using a Fiber Channel to connect your tape drive to Bareos, please be sure to disable any caching in the NSR (network storage router, which is a Fiber Channel to SCSI converter).
```

Comment:
I think Fibre Channel is correct. See https://en.wikipedia.org/wiki/Fibre_Channel

Reply:
Thank for the pointer, its subtil :-)